### PR TITLE
feat: Basic implementation of PieRadarChartTouchListener

### PR DESCRIPTION
**Pull request by DimitrisRK**
## PR Checklist:
- [x] I have tested this extensively and it does not break any existing behavior.
- [ ] I have added/updated examples and tests for any new behavior.
- [ ] If this is a significant change, an issue has already been created where the problem / solution was discussed: [N/A, or add link to issue here]
       

## PR Description
This PR includes:
- Basic implementation of PieRadarChartTouchListener (tap)
- Fix for RadarChart highlight not working on tap
- Removed index.ts. Now, 'charts/index.ts' will be used at package.json
- Small cleaning and organization

### CODE REVIEW

**Comment by farfromrefug** [comment]
@DimitrisRK it is not in sync anymore :s Pushed a ton of changes this morning.
So they are conflicts (i think changes you fixed too).
You might need help understanding my changes (especially the use of `getEntryXValue`). Dont hesitate to contact me on slack

File: src/charting/index.ts [hunk 1]
```diff
@@ -1,5 +0,0 @@
-export { BarChart } from "./charts/BarChart";
```
Comment (farfromrefug) [inline src/charting/index.ts:1]: let s keep this file. As i explained it makes it easier to import charts

**Review by farfromrefug — CHANGES_REQUESTED** [review]

File: src/charting/index.ts (hunk 1 above)
Comment (farfromrefug) [reply src/charting/index.ts:1]: i think i see what you did. Smart! It s related to your change in the typings path?

Comment (DimitrisRK) [reply src/charting/index.ts:1]: @farfromrefug This is a different file. The new one is still in charts/index.ts.
But we can keep this one too, if you want.

**Comment by DimitrisRK** [comment]
> @DimitrisRK it is not in sync anymore :s Pushed a ton of changes this morning.
> So they are conflicts (i think changes you fixed too).
> You might need help understanding my changes (especially the use of `getEntryXValue`). Dont hesitate to contact me on slack

Conflicts resolved. :)

**Pull request merged**